src/app/clusters/occupancy-sensor-server/occupancy-sensor-server.h

```diff
@@ -17,15 +17,48 @@
 
 #pragma once
 
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/AttributeAccessInterface.h>
 #include <app/util/af-types.h>
 #include <app/util/basic-types.h>
+#include <app/util/config.h>
+#include <lib/core/DataModelTypes.h>
 
-/** @brief Occupancy Cluster Server Post Init
- *
- * Following resolution of the Occupancy state at startup for this endpoint,
- * perform any additional initialization needed; e.g., synchronize hardware
- * state.
- *
- * @param endpoint Endpoint that is being initialized  Ver.: always
- */
-void emberAfPluginOccupancyClusterServerPostInitCallback(chip::EndpointId endpoint);
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace OccupancySensing {
+
+class OccupancySensingAttrAccess : public AttributeAccessInterface
```

Comment:
Call it `Instance` to follow the model we have right now in other places. This will be `chip::app::Clusters::OccupancySensing::Instance`.